package.json

```diff
@@ -12,7 +12,7 @@
 		"url": "https://github.com/aws/amazon-states-language-service"
 	},
 	"license": "MIT",
-	"version": "1.5.0",
+	"version": "1.6.0",
```

Comment:
You need to update `package-lock.json` too. 

That can be done manually or through the CLI with `npm version patch`. https://docs.npmjs.com/cli/version

Reply:
Thanks @wong-a, fixed